source/_less/patternfly-site.less

```diff
@@ -1442,3 +1442,40 @@ th {
     }
   }
 }
+
+.pforg-code__title-row {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 25px;
+}
+
+.pforg-code__core-example-title {
+  margin: 0;
+  padding: 0;
+}
+
+.pforg-code__showcase-link-list {
+  list-style: none;
+  margin: 0;
+  padding: 0 0 25px 0;
+  display: flex;
+  flex-wrap: wrap;
+
+  border-bottom: 1px solid #D5D5D5;
```

Comment:
Are the patternfly color variables not available here?

Reply:
Hmm..looking through I don't see a color variable for this. Can double check.

Reply:
black-300 is `#d1d1d1`, pretty close, I'd say that `d5d5d5` is a mistake

Reply:
@andresgalante should I leave the `d5d5d5` or change them all to black-300 because `d5d5d5` is being used in a few different places.

Reply:
@andresgalante I'm not sure what the approach is styling wise when it comes to the site. I'm thinking since the site has already been using #d5d5d5 in the ``patternfly-site.less`` then perhaps I should keep it as is to remain consistent rather than change them all and perhaps deviating from the design?

Reply:
Please file an issue to fix instances where the palette variables are not used.

Reply:
Issue has been filed here #507